src/AccessibilityInsights.SharedUx/Enums/HighlighterColor.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+namespace AccessibilityInsights.SharedUx.Enums
+{
+    /// <summary>
+    /// These values (as strings) correspond to keys of SolidColorBrushes 
+    /// found in the HighlightStyle.xaml resource dictionary
+    /// </summary>
+    public enum HighlighterColor
```

Comment:
>HighlighterColor [](start = 16, length = 16)

Copied from [https://github.com/microsoft/axe-windows/blob/master/src/Core/Enums/HighlighterColor.cs--only](https://github.com/microsoft/axe-windows/blob/master/src/Core/Enums/HighlighterColor.cs--only) change is the namespace